### PR TITLE
Add Dockerfiles for Ubuntu 22.04 LTS (Jammy Jellyfish) Beta

### PR DESCRIPTION
**Pull request by sestegra**
As Ubuntu 22.04 LTS (Jammy Jellyfish) Beta release is available, I suggest to add related Dockerfiles.
https://releases.ubuntu.com/22.04/

The final release is expected on the 21st April.
See [Jammy Jellyfish Release Schedule](https://discourse.ubuntu.com/t/jammy-jellyfish-release-schedule/23906)

### CODE REVIEW

File: ubuntu-jammy-docker-dbg/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#
+# Ubuntu Jammy + Docker
+#
+# Instructions for docker installation taken from:
```
Comment (rodnymolina) [inline ubuntu-jammy-docker-dbg/Dockerfile:4]: This sentence doesn't apply to this file.

**Review by rodnymolina — APPROVED** [review]
LG.

We will generate the corresponding images and have them uploaded to Dockerhub / Github asap. Thanks @sestegra.

File: ubuntu-jammy-docker-dbg/Dockerfile (hunk 1 above)
Comment (sestegra) [reply ubuntu-jammy-docker-dbg/Dockerfile:4]: This sentence is the same for previous versions: ubuntu-bionic-docker-dbg and ubuntu-focal-docker-dbg.
What would you prefer? Change all or keep this as the others.
What message would you like to have here?

Comment (rodnymolina) [reply ubuntu-jammy-docker-dbg/Dockerfile:4]: Oh, I see. Remove it from here please, we'll fix `ubuntu-focal-docker-dbg`, where it shouldn't be, later on.

Comment (sestegra) [reply ubuntu-jammy-docker-dbg/Dockerfile]: Done

**Review by rodnymolina — APPROVED** [review]
LG

**Comment by rodnymolina** [comment]
Thanks a lot @sestegra!

**Pull request merged**